Revised 01/02/2019 'pt-br' Quick-Facts Sheet .md/PDFs (#119)
monero-ecosystem/outreach-docs: monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

# MONERO

> Monero é uma moeda digital segura, privada, não-rastreável que está disponível para todos. As suas características o tornam uma moeda fungível.

Author: Changed

# FATOS RÁPIDOS

> Usuários podem gastar com segurança, ninguém consegue ver seu saldo ou rastrear suas atividades sem sua permissão.

Author: Changed

#### DESCENTRALIZAÇÃO

Sendo um _[projeto código aberto](https://github.com/monero-project/monero)_, liderado e _[financiado](https://ccs.getmonero.org/funding-required/)_ por uma _[equipe descentralizada](https://www.getmonero.org/pt-br/community/team/)_ de desenvolvedores e membros da comunidade, o mesmo não pode ser censurado. A maioria dos contribuidores são voluntários e a comunidade está espalhada por todo o globo.

#### SEGURANÇA

> Sendo uma criptomoeda descentralizada, o Monero é seguro por uma grande rede de usuários espalhados pelo mundo. Transações são confirmadas por um consenso distrubuído e depois gravadas imutavelmente no _[blockchain](https://pt.wikipedia.org/wiki/Blockchain)_.

Author: Changed

#### ESCALABILIDADE

> Diferentemente do Bitcoin e de outras moedas, as quais costumam ter constantes de código inalteráveis (_hardcoded constants_), o _[tamanho do bloco](https://www.mycryptopedia.com/block-size-explained/)_ do Monero é adaptado automaticamente ao volume de transações, garantindo menores taxas e confirmações mais rápidas.

Author: Changed

#### PRIVACIDADE

> Monero usa criptografia sofisticada como _[assinaturas em anel](https://www.getmonero.org/pt-br/resources/moneropedia/ringsignatures.html)_, _[transações confidenciais em anel](https://www.getmonero.org/pt-br/resources/moneropedia/ringCT.html)_, e _[endereços stealth](https://www.getmonero.org/pt-br/resources/moneropedia/stealthaddress.html)_ para ofuscar as origens, as quantidades e os destinatários de todas as transações. Os usuários podem decidir exatamente quais informações serão reveladas e para quem eles querem revelá-las.

Author: Changed

#### RESISTÊNCIA À CENSURA

> Dadas as características de privacidade do Monero, nenhum endereço específico ou carteira podem ser banidos por mineradores ou por agentes econômicos. Desta forma, os usuários ficam livres de censura e do _[controle de capital](https://pt.wikipedia.org/wiki/Controle_de_capital)_.

Author: Changed

 **o:** uma coisa (gramaticamente falando: um substantivo)

> Isso que dizer que "monero" pode ser interpretado como: "um substantivo que descreve a menor parte do dinheiro". Ou, uma moeda.

Reviewer: ```suggestion
Isso quer dizer que "monero" pode ser interpretado como: "um substantivo que descreve a menor parte do dinheiro". Ou, uma moeda.
```

Author: Changed

> **Trajetória das recompensas do bloco do Monero:** As recompensas diminuirão gradativamente até que a emissão em cauda comece por volta do fim de Maio de 2022, quando as recompensas serão fixadas em 0,6 XMR por bloco. A emissão em cauda propocionará incentivos continuados e indefinitivos à mineração. Adicionalmente, e talvez o mais importante, a emissão em cauda dá ao Monero inflação planejada, estável e previsível - considerada essencial para o dinheiro real e saudável.

Reviewer: ```suggestion
**Trajetória das recompensas do bloco do Monero:** As recompensas diminuirão gradativamente até o final de maio de 2022, momento em que ocorrerá a emissão em cauda, que fixará para sempre as recompensas em 0,6 XMR por bloco. A emissão em cauda propocionará incentivos contínuos e eternos à mineração. E, talvez o mais importante, ela fará com que o Monero tenha uma taxa de inflação planejada, estável e previsível para todos - uma característica considerada essencial para uma moeda de qualidade e saudável.
```

Author: Changed

> Embora o Monero já esteja disponível e em uso através do globo, a comunidade de desenvolvedores tem objetivos excitantes para continuar melhorando a privacidade, a segurança e a usabilidade do Monero e das criptomoedas em geral. Estas são algumas das coisas que estão por vir:

Reviewer: ```suggestion
Embora o Monero já esteja disponível e seja usado mundialmente, a comunidade de desenvolvedores tem diversos planos e estratégias para continuar melhorando a privacidade, a segurança e a usabilidade do Monero e das criptomoedas em geral. Estas são algumas das novidades que estão por vir:
```

Author: Changed

> **Kovri:** Esta é uma das maiores melhorias, adicionando uma quarta camada de privacidade às transações do Monero. Kovri utiliza **[criptografia garlic](https://www.getmonero.org/pt-br/resources/moneropedia/garlic-encryption.html)** (uma espécie de Tor turbinado) para esconder o endereço IP dos remetentes e destinatários usando a rede. Todos os elementos que fazem parte de uma transação no Monero, o remetente, destinatário, quantidade transacionada, e (com Kovri) o pilar central da transação em si, serão privados e seguros.

Reviewer: ```suggestion
**Kovri:** Esta é uma das maiores melhorias, adicionando uma quarta camada de privacidade às transações do Monero. Kovri utiliza **[criptografia garlic](https://www.getmonero.org/pt-br/resources/moneropedia/garlic-encryption.html)** (uma espécie de Tor turbinado) para esconder o endereço IP dos remetentes e destinatários usando a rede. Com Kovri, todos os elementos que fazem parte de uma transação no Monero passarão a ser privados e seguros: o remetente (assinaturas em anel), a transmissão da transação/endereço IP (Kovri), o destinatário (endereços stealth) e a quantia transacionada (transações confidenciais em anel).
```

Author: Changed
